docs/org_config-reference.md

```diff
@@ -0,0 +1,59 @@
+# org_config Object Reference
+
+The `org_config` object can be used in the `cumulusci.yml` file to read a large number number of attributes of the Salesforce org currently used. For example, in a [custom flow step](https://cumulusci.readthedocs.io/en/stable/config.html#add-a-flow-step), you can use a `when`clause to adapt the behavior of the new step to the type of org (scratch org or not) by referencing the `org_config.scratch` attribute.
+
+The following information is documented here in the application's source code :
+https://github.com/SFDO-Tooling/CumulusCI/blob/main/cumulusci/core/config/org_config.py
+
+## org_config Object Attributes
+
+-   `access_token` : access token currently used to authenticate with Salesforce
+-   `installed_packages` : comma-separated list of package names; a dict mapping a namespace or package Id (033\*) to the installed package version(s) matching that identifier. All values are lists, because multiple second-generation packages may be installed with the same namespace. Beta version of a package are represented as "1.2.3b5", where 5 is the build number.
```

Comment:
```suggestion
-   `installed_packages` : comma-separated list of package names; a `dict` mapping a namespace or metadata package ID (starts with `033`) to the installed package version(s) matching that identifier. All values are lists, because multiple second-generation packages may be installed with the same namespace. The beta version of a package is represented as "1.2.3b5", where 5 is the build number.
```